### DISCUSSION TITLE
alist部署在服务器上，现在服务器每天会下载一些文件，我如何将文件上传到alist的指定网盘中

### DISCUSSION THREAD

**Discussion by Angel0726** [AlistGo/alist#5200] [Q&A]
aria2本质上是先下载文件到本地，然后上传到alust指定路径中。有没有命令实现上传文件到alist的指定路径中

**Comment by free-150** [comment]
想实现直接下载到网盘中是吧？
在服务器上安装aria2后在 alist 管理的左侧在「设置」「其它」中检查是否成功。
成功后在网盘主页进入你想下载到的网盘目录中，在右下角里有「离线下载」输入 aria2 的链接就能下载到指定网盘目录了。
